RetailCoder.VBE/Common/Hotkeys/Hotkey.cs

```diff
@@ -83,7 +83,8 @@ private void HookKey(Keys key, uint shift)
             var success = User32.RegisterHotKey(_hWndVbe, hookId, shift, (uint)key);
             if (!success)
             {
-                throw new Win32Exception(Rubberduck.UI.RubberduckUI.CommonHotkey_KeyNotRegistered);
+                Debug.WriteLine(Rubberduck.UI.RubberduckUI.CommonHotkey_KeyNotRegistered, key);
+                //throw new Win32Exception(Rubberduck.UI.RubberduckUI.CommonHotkey_KeyNotRegistered, key);
```

Comment:
I'm not sold that this is a good idea ... could we at least `return` here to interrupt execution like before?